lib/trace.js

```diff
@@ -0,0 +1,185 @@
+'use strict'
+
+var Stream = require('stream')
```

Comment:
Ah yeah, I usually try to have as few dependencies in my projects as possible, which is why I rarely use readable-stream, but added it now :)